### PR TITLE
[docs] update docs/usage/webworker

### PR DESCRIPTION
**Pull request by mireille-raad**
This PR has some fixes for the documentation on https://pyodide.readthedocs.io/en/latest/usage/webworker.html

### CODE REVIEW

File: docs/usage/webworker.md [hunk 1]
```diff
@@ -31,22 +31,22 @@ method (and vice versa).
...
 
-* The **web worker** is responsible for running scripts in its own thread separate thread.
+* The **web worker** is responsible for running scripts in its own separate thread.
 * The **worker API** exposes a consumer-to-provider communication interface.
 * The **consumer**s want to run some scripts outside the main thread so they don't block the main thread.
 
 ### Consumers
 
-Our goal is to run some javascript code in another thread, this other thread will
+Our goal is to run some python code in another thread, this other thread will
```
Comment (hoodmane) [inline docs/usage/webworker.md:40]:
```suggestion
Our goal is to run some Python code in another thread, this other thread will
```

File: docs/usage/webworker.md [hunk 2]
```diff
@@ -79,20 +79,20 @@ main();
 ```
 
 Before writing the API, lets first have a look at how the worker operates.
-How does our web worker will run the `script` using a given `context`.
+How does our web worker run the `script` using a given `context`.
 
 ### Web worker
 
-[A worker][worker API] is ...
+Let's start with the definition of [a worker][worker API] is.
```
Comment (hoodmane) [inline docs/usage/webworker.md:86]:
```suggestion
Let's start with the definition. [A worker][worker API] is:
```

**Review by hoodmane — APPROVED** [review]
Thanks!

**Review by rth — APPROVED** [review]
Thanks @mireille-raad !

**Comment by rth** [comment]
There is one merge conflict now, could you please merge master in to resolve it?

**Comment by mireille-raad** [comment]
merge conflict fixed. It was from another PR that was normalizing the capitalization of Python and Javascript. 

Good to merge?

**Comment by rth** [comment]
yes, thanks!

**Pull request merged**